subprojects/core-api/src/main/java/org/gradle/api/initialization/SharedModelDefaults.java

```diff
@@ -31,6 +32,15 @@
 @Incubating
 @ServiceScope(Scope.Build.class)
 public interface SharedModelDefaults {
+
+    /**
+     * Provides access to the project layout when default configurations are
+     * evaluated in the context of specific projects.
+     *
+     * @since 8.14
```

Comment:
💭 It might be good to document the behavior of accessing the property outside of the project context.

Reply:
Added some clarification, not 100% sure it's what you meant.

Reply:
Yep, exactly that: some explicit warning about using the property outside of the software type defaults.